pkg/watch/watch.go

```diff
@@ -38,13 +35,26 @@ import (
 const (
 	// PushErrorString is the string that is printed when an error occurs during watch's Push operation
 	PushErrorString = "Error occurred on Push"
-	CtrlCMessage    = "Press Ctrl+c to exit `odo dev` and delete resources from the cluster"
+	PromptMessage   = `
+[Ctrl+c] - Exit and delete resources from the cluster
+     [p] - Manually sync / push files to the cluster
```

Comment:
One last thing I was thinking of: maybe we could also say that pressing 'p' might also restart/reload the running application.
This could be mentioned here and/or in the documentation for `odo dev`.
Other than sync'ing source files, I didn't see anything related to the running application mentioned (unless that's what we mean by "sync").